pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -505,6 +524,44 @@ func (r *ReconcileIBMBlockCSI) reconcileServiceAccount(instance *ibmblockcsi.IBM
 	return nil
 }
 
+func (r *ReconcileIBMBlockCSI) getRestartedAtAnnotation(Annotations map[string]string) (string, string){
+	restartedAt := fmt.Sprintf("%s/restartedAt", oconfig.APIGroup)
+	for key, element := range Annotations {
+		if key == restartedAt {
+			return key, element
+		}
+	}
+	return "", ""
+}
+
+func (r *ReconcileIBMBlockCSI) getControllerK8sObject(instance *ibmblockcsi.IBMBlockCSI) (*appsv1.StatefulSet, error) {
+			controllerStatefulset := &appsv1.StatefulSet{}
+			err := r.client.Get(context.TODO(), types.NamespacedName{
+				Name:      oconfig.GetNameForResource(oconfig.CSIController, instance.Name),
+				Namespace: instance.Namespace,
+			}, controllerStatefulset)
+
+			return controllerStatefulset, err
+}
+
+func (r *ReconcileIBMBlockCSI) getNodeK8sObject(instance *ibmblockcsi.IBMBlockCSI) (*appsv1.DaemonSet, error) {
+	node := &appsv1.DaemonSet{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{
+		Name:      oconfig.GetNameForResource(oconfig.CSINode, instance.Name),
+		Namespace: instance.Namespace,
+	}, node)
+
+	return node, err
+}
```

Comment:
rename `k8sObject` to the specific k8s object, and fix over indentation

Reply:
done

Reply:
no need for the word `Object`

Reply:
done